src/kernel/include/pgtbl.h

```diff
@@ -47,6 +49,42 @@ typedef enum {
 #define PGTBL_DEPTH 2
 #define PGTBL_ORD 10
 
+/* The page size definitions for composite OS */
+#define PGSZ_128B	7
```

Comment:
Guess the CGS suggests to use `enums` in such cases. I personally prefer enumerations here!

Reply:
Will change to enums